src/main/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAO.java

```diff
@@ -58,30 +58,29 @@ List<DarCollectionSummary> getDarCollectionSummariesForDAC(
   @UseRowReducer(DarCollectionSummaryReducer.class)
   @SqlQuery
       (
-          "SELECT c.collection_id as dar_collection_id, c.dar_code, dar.submission_date, dar.reference_id as dar_reference_id, u.display_name as researcher_name, "
-              +
-              "i.institution_name, e.election_id, e.status, e.dataset_id, e.reference_id, dd.dataset_id as dd_datasetid, "
-              +
-              "(regexp_replace(dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb ->> 'projectTitle' AS name " +
-              "FROM dar_collection c " +
-              "INNER JOIN users u " +
-              "ON u.user_id = c.create_user_id " +
-              "LEFT JOIN institution i " +
-              "ON i.institution_id = u.institution_id " +
-              "INNER JOIN data_access_request dar " +
-              "ON dar.collection_id = c.collection_id " +
-              "LEFT JOIN ( " +
-              "SELECT election.*, MAX(election.election_id) OVER(PARTITION BY election.reference_id, election.dataset_id) AS latest "
-              +
-              "FROM election " +
-              "WHERE LOWER(election.election_type) = 'dataaccess'" +
-              ") AS e " +
-              "ON e.reference_id = dar.reference_id " +
-              "INNER JOIN dar_dataset dd " +
-              "ON dar.reference_id = dd.reference_id " +
-              "WHERE u.institution_id = :institutionId " +
-              "AND (e.latest = e.election_id OR e.election_id IS NULL) " +
-              "AND (LOWER(data->>'status') != 'archived' OR data->>'status' IS NULL ) "
+          """
+              SELECT c.collection_id as dar_collection_id, c.dar_code, dar.submission_date, dar.reference_id as dar_reference_id, u.display_name as researcher_name,
+              i.institution_name, e.election_id, e.status, e.dataset_id, e.reference_id, dd.dataset_id as dd_datasetid,
+              (regexp_replace(dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb ->> 'projectTitle' AS name
+              FROM dar_collection c
+              INNER JOIN users u
+              ON u.user_id = c.create_user_id
+              LEFT JOIN institution i
+              ON i.institution_id = u.institution_id
+              INNER JOIN data_access_request dar
+              ON dar.collection_id = c.collection_id
+              LEFT JOIN (
+              SELECT election.*, MAX(election.election_id) OVER(PARTITION BY election.reference_id, election.dataset_id) AS latest
+              FROM election
+              WHERE LOWER(election.election_type) = 'dataaccess'
+              ) AS e
```

Comment:
Minor nit: it would be nice to add a level of indentation to the inner select. Previous code didn't do that, but as a text block, it would be a lot more legible. Similar comment applies to other query reformats in this class.